src/main/java/frc/robot/subsystems/hood/commands/HoodCommand.java

```diff
@@ -2,32 +2,30 @@
 
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.Constants;
-import frc.robot.RobotContainer;
 import frc.robot.subsystems.hood.Hood;
 
 public class HoodCommand extends CommandBase {
```

Comment:
You can remove the class, there is no need to just comment classes as it would make more troubles  for other commands who use this command...